src/main/java/com/alkemy/ong/service/impl/UserServiceImpl.java

```diff
@@ -4,43 +4,51 @@
 import com.alkemy.ong.dto.UserPatchDTO;
 import com.alkemy.ong.dto.UserResponseDto;
 import com.alkemy.ong.exception.UserNotFoundException;
+import com.alkemy.ong.mail.EmailService;
 import com.alkemy.ong.mapper.UserMapper;
 import com.alkemy.ong.mapper.UserResponseMapper;
 import com.alkemy.ong.model.User;
 import com.alkemy.ong.repository.UserRepository;
 import com.alkemy.ong.service.UserService;
 import lombok.RequiredArgsConstructor;
+import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.security.crypto.password.PasswordEncoder;
 import org.springframework.stereotype.Service;
 import org.springframework.transaction.annotation.Transactional;
 
 @Service
 @RequiredArgsConstructor
 public class UserServiceImpl implements UserService {
-  
+
     private final UserRepository userRepository;
     private final PasswordEncoder passwordEncoder;
     private final UserMapper userMapper;
     private final UserResponseMapper userResponseMapper;
 
+    @Autowired
```

Comment:
Could you please use the same dependency injection strategy, removing @Autowired and making the attribute emailService final like:

`private final EmailService emailService;`

(This is possible because de lombok's RequiredArgsConstructor decorator)